static/src/js/containers/SavedProjectsContainer/SavedProjectsContainer.jsx

```diff
@@ -1,25 +1,14 @@
 import React from 'react'
-import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
 
-import actions from '../../actions'
-
 import SavedProjects from '../../components/SavedProjects/SavedProjects'
 
-export const mapDispatchToProps = (dispatch) => ({
-  onHandleError: (errorConfig) => dispatch(actions.handleError(errorConfig))
-})
+export const mapDispatchToProps = () => ({})
 
-export const SavedProjectsContainer = ({
-  onHandleError
-}) => (
-  <SavedProjects
-    onHandleError={onHandleError}
-  />
+export const SavedProjectsContainer = () => (
+  <SavedProjects />
```

Comment:
Look at where we are calling SavedProjectsContainer, and just call SavedProjects